libs/application/templates/financial-aid/src/fields/ConfirmationSectionImage/ConfirmationSectionImage.tsx

```diff
@@ -0,0 +1,3740 @@
+        fill="#6A2EA0"
+      />
+      <path
+        d="M548.814 138.774C548.478 138.774 548.207 138.503 548.207 138.167V118.35C548.207 118.014 548.478 117.743 548.814 117.743C549.15 117.743 549.421 118.014 549.421 118.35V138.167C549.421 138.503 549.15 138.774 548.814 138.774Z"
+        fill="#6A2EA0"
+      />
+      <path
+        d="M496.648 146.356C496.587 146.356 496.526 146.347 496.47 146.327C480.344 141.377 472.381 136.374 472.304 136.325C472.02 136.147 471.935 135.771 472.118 135.488C472.296 135.204 472.672 135.119 472.955 135.301C473.032 135.35 480.826 140.24 496.672 145.121C499.415 144.49 536.057 136.208 558.94 137.73C559.276 137.754 559.527 138.042 559.506 138.378C559.486 138.714 559.191 138.965 558.859 138.944C535.381 137.382 497.174 146.25 496.789 146.339C496.741 146.352 496.692 146.356 496.648 146.356Z"
+        fill="#6A2EA0"
+      />
+      <path
+        d="M457.998 151.962C457.743 151.962 457.508 151.804 457.423 151.549C457.314 151.233 457.488 150.885 457.804 150.78C457.901 150.747 467.799 147.42 478.668 145.158C478.999 145.089 479.319 145.299 479.384 145.627C479.453 145.955 479.242 146.279 478.914 146.344C468.118 148.59 458.289 151.893 458.192 151.925C458.127 151.95 458.063 151.962 457.998 151.962Z"
+        fill="#6A2EA0"
+      />
+      <path
+        d="M466.662 164.833C466.374 164.833 466.12 164.631 466.067 164.339C466.006 164.012 466.221 163.692 466.552 163.631L496.537 158.025C496.574 158.017 496.61 158.013 496.651 158.013H586.565C586.5 155.163 585.998 142.68 582.117 128.396C578.067 113.497 569.403 93.5214 551.121 82.9893C543.76 78.7474 537.614 77.1121 533.757 76.4847C529.581 75.8088 527.109 76.1366 527.085 76.1407C526.753 76.1892 526.445 75.9545 526.401 75.6226C526.352 75.2907 526.587 74.983 526.915 74.9385C527.327 74.8818 537.245 73.5947 551.728 81.9369C570.378 92.6795 579.192 112.974 583.299 128.105C587.738 144.457 587.791 158.478 587.791 158.616C587.791 158.952 587.519 159.223 587.184 159.223H496.711L466.783 164.821C466.735 164.829 466.698 164.833 466.662 164.833Z"
+        fill="#6A2EA0"
+      />
+      <path
+        d="M561.189 245.39C560.853 245.39 560.582 245.119 560.582 244.783V159.276C560.582 158.94 560.853 158.669 561.189 158.669C561.525 158.669 561.796 158.94 561.796 159.276V244.783C561.796 245.119 561.525 245.39 561.189 245.39Z"
+        fill="#6A2EA0"
+      />
+      <path
+        d="M391.287 245.39C390.967 245.39 390.7 245.139 390.68 244.815C390.668 244.609 389.721 223.836 402.589 210.26C410.006 202.436 420.503 198.465 433.788 198.46
```

Comment:
ehh.... are you sure about this? Let's talk about this IRL

Reply:
btw I noticed it is the exactly the same image, where could I possibly upload the image?